src/core/MOM_barotropic.F90

```diff
@@ -1598,15 +1593,9 @@ subroutine btstep(U_in, V_in, eta_in, dt, bc_accel_u, bc_accel_v, forces, pbce,
     endif ; enddo ; enddo
   endif
 
-  ! Here is an example of how the filter equations are time stepped to determine the M2 and K1 velocities.
-  ! The filters are initialized and registered in subroutine barotropic_init.
-  if (CS%use_filter_m2) then
-    call Filt_accum(ubt, um2, CS%Time, US, CS%Filt_CS_um2)
-    call Filt_accum(vbt, vm2, CS%Time, US, CS%Filt_CS_vm2)
-  endif
-  if (CS%use_filter_k1) then
-    call Filt_accum(ubt, uk1, CS%Time, US, CS%Filt_CS_uk1)
-    call Filt_accum(vbt, vk1, CS%Time, US, CS%Filt_CS_vk1)
+  if (CS%use_filter) then
```

Comment:
The barotropic solver is called twice, once in the predictor step and once in the corrector step.  For other diagnostics in MOM_barotropic.F90, we use a call to `query_averaging_enabled()` to figure out whether we are in the predictor step or the corrector step, and then we only diagnose what is happening on the predictor step.  To help with any confusion when we are making use of `ufilt` and `vfilt` in future additions of diagnostics or tide-specific parameterizations, I think that it would be very helpful if we were to add a comment here explaining this, such as:
`! Note that the filtered velocities are only updated by filt_accum during the predictor step btstep call.`